tabular/src/autogluon/tabular/models/tabular_nn/tabular_nn_model.py

```diff
@@ -191,10 +191,6 @@ def _fit(self, X_train, y_train, X_val=None, y_val=None, time_limit=None, report
                len(train_dataset.feature_groups['language']) ))
         # self._save_preprocessor()  # TODO: should save these things for hyperparam tunning. Need one HP tuner for network-specific HPs, another for preprocessing HPs.
 
-        if 'num_gpus' in kwargs and kwargs['num_gpus'] >= 1:
```

Comment:
Why is this deleted?

Reply:
I think previously I removed it to try to make tabular nn work but let me add back.

Reply:
Added back.